src/main/kotlin/net/maizegenetics/phgv2/cli/CreateMafVcf.kt

```diff
@@ -563,7 +546,7 @@ class CreateMafVcf : CliktCommand(help = "Create g.vcf and h.vcf files from Anch
                 "<ID=${assemblyHaplotypeHash}, Description=\"haplotype data for line: ${metaDataRecord.sampleName}\">," +
                         "Source=\"${dbPath}/assemblies.agc\",SampleName=\"${metaDataRecord.sampleName}\"," +
                         "Regions=\"${metaDataRecord.asmRegions.map { "${it.first.contig}:${it.first.position}-${it.second.position}" }.joinToString(",")}\"," +
-                        "Checksum=\"Md5\",RefRange=\"${refSeqHash}\">",
+                        "Checksum=\"${assemblyHaplotypeHash}\",RefChecksum=\"${refSeqHash}\",RefRange=\"${metaDataRecord.refContig}:${metaDataRecord.refStart}-${metaDataRecord.refEnd}\">",
```

Comment:
Did we decide not to store the means of calculating the check sum (Md5, SHA-256, etc)?

Reply:
That was my understand?